README.md

```diff
@@ -2,13 +2,58 @@
 
 This app is built using [SvelteKit](https://kit.svelte.dev/) and [TypeScript](https://www.typescriptlang.org/).
 
+## Prerequisites
+
+For development, you need the following installed:
+
+- [Docker](https://docs.docker.com/engine/install/)
+- [Supabase CLI](https://supabase.com/docs/guides/cli)
```

Comment:
it looks like we can install `supabase` with npm if we want that to live in our `package.json` and not as pre-req dependency: https://supabase.com/docs/guides/cli#installation

Reply:
Yeah I probably should've done that but I made the decision to brew install it. I can switch to that though.

Reply:
Added in [1a8b60a](https://github.com/johnverrone/bunkie-bois/pull/8/commits/1a8b60a57d697f9f20091afec689af8a4185731e)